compiler/rustc_lint/messages.ftl

```diff
@@ -496,6 +496,10 @@ lint_requested_level = requested on the command line with `{$level} {$lint_name}
 
 lint_span_use_eq_ctxt = use `.eq_ctxt()` instead of `.ctxt() == .ctxt()`
 
+lint_static_mut_ref = use of mutable static
```

Comment:
this probably should say something more..

Reply:
any suggestion?

Reply:
"is discouraged" or something... right now it's literally just saying "use if mutable static". My thought is immediately "ok, so what?"

This should also include a note saying why, and also link to something more that explains why mutable statics should not be used.